### ISSUE TITLE
Kafka: Increase session timeout for zookeeper connection

### ISSUE THREAD

**Issue by manalibhutiyani** [cilium/cilium#3157]
The standard session timeout of 6s is often not long enough when the broker
starts. Increasing this to 20s increases the reliability.

**Comment by manalibhutiyani** [comment]
We need to change the yaml to use this:

```
        - name: KAFKA_CREATE_TOPICS
            value: "empire-announce:1:1,deathstar-plans:1:1"
        - name: KAFKA_ZOOKEEPER_SESSION_TIMEOUT_MS
          value: "20000"
```

**Comment by manalibhutiyani** [comment]
Fixed via #3286
